doc/appendix-de-kassensichv/operation-modes/on-premise-platforms/android.md

```diff
@@ -16,47 +16,47 @@ The launcher of the fiskaltrust.Middleware for Android can be used with followin
+The middleware background service must be started via Android intent and then, as usual, make REST or gRPC endpoints available via our standard interface.
 
 ## Distribution
 
 ### fiskaltrust.Portal
 
-Die Android Launcher APK-files werden jeweils für die HTTP-Variante als auch für die gRPC Variante im fiskaltrust.Portal zum Download angeboten, wenn die oben beschriebenen Voraussetzungen erfüllt sind.
+The Android Launcher APK files are offered for download in the fiskaltrust.Portal for the HTTP variant and the gRPC variant if the requirements described above are met.
 
 ### Google Play Store
 
-Der Android Launcher wird jeweils für die HTTP-Variante als auch für die gRPC Variante im Google Play Store als App zum Download angeboten, wenn die oben beschriebenen Voraussetzungen erfüllt sind.
+The Android Launcher is available for download as an app for both the HTTP variant and the gRPC variant in the Google Play Store if the requirements described above are met.
 
-|                         | fiskaltrust.Middleware für Android (HTTP)                    | fiskaltrust.Middleware für Android (gRPC)                    |
-| ----------------------- | ------------------------------------------------------------ | ------------------------------------------------------------ |
-| **QR Code zum Scannen** | ![http](../images/android-http.png)                          | ![grpc](../images/android-grpc.png)                          |
-| **URL**                 | https://play.google.com/store/apps/details?id=eu.fiskaltrust.androidlauncher.http | https://play.google.com/store/apps/details?id=eu.fiskaltrust.androidlauncher.grpc |
+|              | fiskaltrust.Middleware for Android (HTTP)                    | fiskaltrust.Middleware for Android (gRPC)                    |
+| ------------ | ------------------------------------------------------------ | ------------------------------------------------------------ |
+| **QR Code ** | ![http](../images/android-http.png)                          | ![grpc](../images/android-grpc.png)                          |
+| **URL**      | https://play.google.com/store/apps/details?id=eu.fiskaltrust.androidlauncher.http | https://play.google.com/store/apps/details?id=eu.fiskaltrust.androidlauncher.grpc |
 
 
 
 ### Troubleshooting
 
 #### Status
 
-Der Status der fiskaltrust.Middleware wird im Android-Benachrichtigungsbereich angezeigt. Alternativ stehen HTTP-Endpunkte zur Verfügung, um den Status und detaillierte Diagnose-Logs abzurufen (s. *Weiterführende Informationen*).
+The status of the fiskaltrust.Middleware is displayed in the Android notification area. Alternatively, HTTP endpoints are available to call up the status and detailed diagnostic logs (see *Additional information*).
 
-#### Initialisierung der Swissbit TSE
+#### Initialisation of the Swissbit TSE
```

Comment:
```suggestion
#### Initialisation of the Swissbit TSE (microSD)
```

Reply:
is the Android support of Swissbit limited to microSD format only?